### PR TITLE
View an article

### PR DESCRIPTION
**Pull request by SamPetcher**

### CODE REVIEW

File: src/components/ArticleCard.jsx [hunk 1]
```diff
@@ -1,4 +1,6 @@
+import { Outlet, Link } from "react-router-dom";
```
Comment (roisinoreilly) [inline src/components/ArticleCard.jsx:1]: are you using outlet yet?

File: src/components/Article.jsx [hunk 2]
```diff
@@ -1,6 +1,51 @@
...
+        return "💻";
+      case "football":
+        return "⚽";
+      default:
+        return "O";
+    }
+  }
+function ArticleView({ article }) {
+  const [articleBody, setArticleBody] = useState([]);
+  const [isLoaded, setIsLoaded] = useState(false);
```
Comment (roisinoreilly) [inline src/components/Article.jsx:17]: you're setting a loading state but not actually using it to display a loading message to a user yet when loading a single article

**Review by roisinoreilly — APPROVED** [review]
- as much as I love your colour palette, Waze is saying your light grey text isn't enough of a contrast against your background :( try making either the text darker or the background lighter to keep it accessible
- I love how you're displaying a single article looks very slick
- you're not using your loading state in your article.jsx to display a loading screen to the user

looking really great though keep it up! merge when ready :)

File: src/components/ArticleCard.jsx (hunk 1 above)
Comment (SamPetcher) [reply src/components/ArticleCard.jsx:1]: > 

Thanks for the catch on grey. Completely missed that! Will sort out loading as well :)

**Pull request merged**